# Optitrack rigid bodies
Header                  header
optitrack/RigidBody[]  bodies
